src/core/openthread-core-default-config.h

```diff
@@ -416,6 +416,16 @@
 #endif  // OPENTHREAD_CONFIG_STORE_FRAME_COUNTER_AHEAD
 
 /**
+  * @def OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE
+  *
+  * Define to 1 if you want use legacy transmit done.
+  *
+  */
+#ifndef OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE
```

Comment:
I guess plan of action is to remove this feature fully from source code when all radio driver converts to new tx done callback model.  I am ok with not having this there :)